whisper/cli/src/main.rs

```diff
@@ -68,7 +83,8 @@ Options:
 	-s, --secret KEYFILE           Specify which file contains the key to generate the enode.
 	-P, --rpc-port PORT            Specify which RPC port to use [default: 8545].
 	-A, --rpc-address ADDRESS      Specify which RPC address to use [default: 127.0.0.1].
-	-l, --log LEVEL                Specify the logging level. Must conform to the same format as RUST_LOG [default: Error].
+	-l, --log-level LEVEL          Specify the logging level. Must conform to the same format as RUST_LOG [default: 4]. Log level verbosity configuration choice from highest to lowest priority (error 0, warn 1, info 2, debug 3, trace 4).
```

Comment:
If we shall change these, it should be the same as in `parity-ethereum` IMHO

https://github.com/paritytech/parity-ethereum/blob/master/parity/cli/mod.rs#L851-#L858

Reply:
`trace!` is definitely too verbose to be enabled by default

Reply:
Ok, I just had a look at that. I think `--logging` sound more like it takes a boolean. But parity-ethereum would get used a lot, so changing that flag to `--log-level` instead wouldn't add much value but it may require a lot of people to have to change their scripts, which may be annoying. So I'll change it to `--logging` instead of `--log-level` and you've suggested

Reply:
I'll change it back to `error!` level logging by default.

Reply:
@niklasad1 I've pushed two commits in this PR addressing your two comments (i.e. switch to default level of Error (0) and rename `--log-level` to `--logging`.

I've further updated another similar PR for evmbin for consistency with parity-ethereum and this PR in this commit https://github.com/paritytech/parity-ethereum/pull/10761/commits/4d9b74477362394ceebc14583545dc196328378b